README.rst

```diff
@@ -3,7 +3,7 @@ CircuitPython Weekly Meeting Notes
 
 Each week, members of the CircuitPython community meet on `the Adafruit Discord <http://adafru.it/discord>`_ to discuss all things CircuitPython. The meetings are recorded and made `available on YouTube <https://www.youtube.com/playlist?list=PLjF7R1fz_OOUvw7tMv45xjWp0ht8yNgg0>`_ with links to notes in this repo. These notes include time codes to specific sections of the video so that one can only listen to portions of interest.
 
-The weekly happens normally at 2pm ET/11am PT on Mondays. Check the #circuitpython channel for notices of change in time and links to past episodes.
+The weekly happens normally at 2pm ET/11am PT on Mondays. Check the #circuitpython channel for notices of change in time and links to past episodes.  You can also subscribe to the meeting calendar `calendar.ical <`https://raw.githubusercontent.com/adafruit/{{ full_repo_name }}/master/calendar.ical>`_.
```

Comment:
I don't think this is correct either. The filename below is `meeting.ical`. (Thanks for doing this! I'll subscribe myself once it is merged.)